stunnel/init.sls

```diff
@@ -21,18 +21,27 @@ stunnel_package:
     - user: {{ stunnel.root_user }}
     - group: {{ stunnel.group }}
     - mode: 750
+{%- if stunnel.lookup.clean_conf_dir %}
+    - clean: True
+{%- endif %}
 
 {{ stunnel.conf_dir }}/services.d:
   file.directory:
     - user: {{ stunnel.root_user }}
     - group: {{ stunnel.group }}
     - mode: 750
+{%- if stunnel.lookup.clean_conf_dir %}
+    - clean: True
+{%- endif %}
 
 {{ stunnel.conf_dir }}/tls:
   file.directory:
     - user: {{ stunnel.root_user }}
     - group: {{ stunnel.group }}
     - mode: 750
+{%- if stunnel.lookup.clean_conf_dir %}
```

Comment:
and here

Reply:
also, this folder is populated in another state `pillar_certs.sls`, so if you purge it in this state, bad things will happen if people don't run `pillar_certs.sls` after `init.sls`